src/About/style.scss

```diff
@@ -0,0 +1,86 @@
+@import '../variables/_variables.scss';
+
+.wrapper-about {
+  background: fixed url("../image/bg_about.png") no-repeat center center / cover;
+}
+
+.section-about {
+  display: flex;
+  width: 70%;
+  padding: 20px 0 20px;
```

Comment:
I believe linter disagrees with u about that

Reply:
yes,changed